test/parallel/test-http-uri-overflow.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+const assert = require('assert');
+const { createServer, maxUriSize } = require('http');
+const { createConnection } = require('net');
+const { expectsError, mustCall } = require('../common');
+
+const CRLF = '\r\n';
+const REQUEST_METHOD_AND_SPACE = 'GET ';
+const DUMMY_URI = '/' + 'a'.repeat(
+  maxUriSize
+); // the slash makes it just 1 byte too long
```

Comment:
Nit: The linter is not happy with this.  Please capitalize the first word of the comment and add a dot at the end.